Source/Tools/FEXLoader/CMakeLists.txt

```diff
@@ -1,148 +1,156 @@
+    # Enable FEX APIs to be used by targets that use target_link_libraries on FEXLoader
+    set_target_properties(${NAME} PROPERTIES ENABLE_EXPORTS 1)
 
-  if (CMAKE_BUILD_TYPE MATCHES "RELEASE")
-    target_link_options(${NAME}
+    target_include_directories(${NAME}
       PRIVATE
-        "LINKER:--gc-sections"
-        "LINKER:--strip-all"
-        "LINKER:--as-needed"
-    )
-  endif()
-
-  if (NOT MINGW_BUILD)
-    install(TARGETS ${NAME}
-      RUNTIME
-        DESTINATION bin
-        COMPONENT runtime
-    )
-  endif()
-endfunction()
-
-GenerateInterpreter(FEXLoader 0)
-GenerateInterpreter(FEXInterpreter 1)
-
-install(PROGRAMS "${PROJECT_SOURCE_DIR}/Scripts/FEXUpdateAOTIRCache.sh" DESTINATION bin RENAME FEXUpdateAOTIRCache)
-
-if (CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
-  # Check for conflicting binfmt before installing
-  set (CONFLICTING_BINFMTS_32
-    ${CMAKE_INSTALL_PREFIX}/share/binfmts/qemu-i386
-    ${CMAKE_INSTALL_PREFIX}/share/binfmts/box86)
-  set (CONFLICTING_BINFMTS_64
-    ${CMAKE_INSTALL_PREFIX}/share/binfmts/qemu-x86_64
-    ${CMAKE_INSTALL_PREFIX}/share/binfmts/box64)
-
-  find_program(UPDATE_BINFMTS_PROGRAM update-binfmts)
-  if (UPDATE_BINFMTS_PROGRAM)
-    add_custom_target(binfmt_misc_32
-      echo "Attempting to install FEX-x86 misc now."
-      COMMAND "${CMAKE_SOURCE_DIR}/Scripts/CheckBinfmtNotInstall.sh" ${CONFLICTING_BINFMTS_32}
-      COMMAND "update-binfmts" "--importdir=${CMAKE_INSTALL_PREFIX}/share/binfmts/" "--import" "FEX-x86"
-      COMMAND ${CMAKE_COMMAND} -E
-      echo "binfmt_misc FEX-x86 installed"
+        ${CMAKE_CURRENT_SOURCE_DIR}/
+        ${CMAKE_BINARY_DIR}/generated
     )
-
-    add_custom_target(binfmt_misc_64
-      COMMAND ${CMAKE_COMMAND} -E
-      echo "Attempting to install FEX-x86_64 misc now."
-      COMMAND "${CMAKE_SOURCE_DIR}/Scripts/CheckBinfmtNotInstall.sh" ${CONFLICTING_BINFMTS_64}
-      COMMAND "update-binfmts" "--importdir=${CMAKE_INSTALL_PREFIX}/share/binfmts/" "--import" "FEX-x86_64"
-      COMMAND ${CMAKE_COMMAND} -E
-      echo "binfmt_misc FEX-x86_64 installed"
+    target_link_libraries(${NAME}
+      PRIVATE
+        ${LIBS}
+        LinuxEmulation
+        ${PTHREAD_LIB}
+        fmt::fmt
     )
-    if(TARGET uninstall)
-      add_custom_target(uninstall_binfmt_misc_32
-        COMMAND update-binfmts --unimport FEX-x86 || (exit 0)
-      )
-      add_custom_target(uninstall_binfmt_misc_64
-        COMMAND update-binfmts --unimport FEX-x86_64 || (exit 0)
+    target_compile_definitions(${NAME} PRIVATE -DFEXLOADER_AS_INTERPRETER=${AsInterpreter})
+
+    if (CMAKE_BUILD_TYPE MATCHES "RELEASE")
+      target_link_options(${NAME}
+        PRIVATE
+          "LINKER:--gc-sections"
+          "LINKER:--strip-all"
+          "LINKER:--as-needed"
       )
+    endif()
 
-      add_dependencies(uninstall uninstall_binfmt_misc_32)
-      add_dependencies(uninstall uninstall_binfmt_misc_64)
+    if (NOT MINGW_BUILD)
```

Comment:
This is already checked in the outer scope

Reply:
Resolved.